components/intake.py

```diff
@@ -22,7 +22,7 @@ class IntakeComponent:
     MOTOR_RPM_TO_SHAFT_RAD_PER_SEC = MOTOR_REV_TO_SHAFT_RADIANS / 60
 
     SHAFT_REV_RETRACT_HARD_LIMIT = 0.0
-    SHAFT_REV_DEPLOY_HARD_LIMIT = 1.8675022996
```

Comment:
This does not make sense as a change on its own, there was a change in the gear ratio which should be included as a part of this

Reply:
This assessment was incorrect. It seems the ratio was already updated